src/Lecture1.hs

```diff
@@ -32,14 +32,16 @@ module Lecture1
     ) where
 
 -- VVV If you need to import libraries, do it after this line ... VVV
+import Data.Char(digitToInt)
```

Comment:
I think this import is no longer needed in this module, so you can safely remove it 🙂